app/search/-components/filter-facet/template.hbs

```diff
@@ -0,0 +1,106 @@
+            @layout='fake-link'
+            local-class='facet-expand-button'
+            aria-controls={{facetElementId}}
+            aria-expanded={{not this.collapsed}}
+            {{on 'click' (fn this.toggleFacet this.label)}}
+        >
+            <span>{{@propertyRecord.label}}</span>
+            <FaIcon @icon={{if this.collapsed 'caret-down' 'caret-up'}} />
+        </Button>
+        {{#if this.fetchFacetValues.isRunning}}
+            <LoadingIndicator @dark={{true}} />
+        {{else if this.fetchFacetValues.isError}}
+            {{t 'search.filter-facet.facet-load-failed'}}
+        {{else}}
+            <ul
+                local-class='facet-list {{if this.collapsed 'collapsed'}}'
+                id={{facetElementId}}
+            >
+                {{#each this.filterableProperties as |property|}}
+                    <li
+                        data-test-filter-facet-value={{property.metadataRecord.title}}
+                        local-class='facet-value'
+                    >
+                        <OsfLink
+                            data-analytics-name='toggle filter facet value {{property.metadataRecord.title}}'
+                            local-class='facet-link'
+                            @route='search'
+                            @query={{hash q=this.q filter=this.filter}}
+                            {{on 'click' (fn @toggleFilter (hash property=@propertyRecord.label value=property.metadataRecord.title))}}
+                        >
+                            {{property.metadataRecord.title}}
+                        </OsfLink>
+                        <span local-class='facet-count'>
+                            {{property.recordResultCount}}
+                        </span>
+                    </li>
+                {{/each}}
+               
+                {{#if this.showSeeMoreButton}}
+                    <li>
+                        <Button
+                            data-analytics-name='see more filterable values {{@propertyRecord.label}}'
+                            data-test-see-more-filterable-values={{@propertyRecord.label}}
+                            @layout='fake-link'
+                            {{on 'click' (fn (mut this.seeMoreModalShown) true)}}
+                        >
+                            {{t 'search.filter-facet.see-more'}}
+                        </Button>
+                    </li>
+                {{/if}}
+            </ul>
+        {{/if}}
+        <OsfDialog
+            data-analytics-scope='filter facet - see more modal'
+            @isOpen={{this.seeMoreModalShown}}
+            @onClose={{fn (mut this.seeMoreModalShown) false}}
+            as |dialog|
+        >
+            <dialog.heading data-test-see-more-dialog-heading>
+                {{@propertyRecord.label}}
+            </dialog.heading>
+            <dialog.main local-class='see-more-dialog'>
+                {{t 'search.filter-facet.see-more-modal-text'}}
+                <PowerSelect
```

Comment:
So 'See more' always brings up a PowerSelect and not different things for different facets now? That's good.

Reply:
Yea, that's the way I have it now. There may be some properties that are kind of weird as a power-select, so that _may_ change, but that's more of a "future improvement"